package.json

```diff
@@ -200,6 +200,19 @@
           },
           "default": null,
           "description": "Development setting: The PPE environment for testing."
+        },
+        "azure.authenticationLibrary": {
+          "type": "string",
+          "enum": [
+            "ADAL",
+            "MSAL (Preview)"
```

Comment:
I think "Preview" is more of a human-readable description than a programmatic value, so I would only include it in "enumDescriptions", not "enum". For example, eventually "MSAL" won't be preview anymore, but we still want a user's hard-coded settings to continue working